src/components/stack/index.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+const Stack = ({
+	additionalClassNames,
+	alignment,
+	children,
+	direction,
+	divider,
+	flexType,
+	justification,
+	gap,
+	wrap,
+}) => {
+	const defaultAndAdditionalClassnames = `c-stack${additionalClassNames ? ` ${additionalClassNames}` : ''}`;
+	return (
+		<div
+			className={defaultAndAdditionalClassnames}
+			data-style-direction={direction}
+			data-style-justification={justification}
+			data-style-alignment={alignment}
+			data-style-flextype={flexType}
+			data-style-wrap={wrap}
+			style={{ '--c-stack-gap': gap }}
+		>
+			{ Array.isArray(children)
+				? children.map((child) => (
+					<Fragment key={child.key}>
+						{ child }
+						{divider && <hr aria-hidden="true" />}
```

Comment:
The divider is to only be output _between_ children elements - we will need to add logic to ensure one is not added to the bottom

Reply:
OK, makes sense